src/data/dynamic/form/Admins.js

```diff
@@ -13,7 +13,7 @@ export const FIELDS = {
     input: "description",
     width: 12,
     texts: [
-      "Welcome to HACKATHON_NAME. Thank you for being on the organizing team, we appreciate your efforts to help support HACKATHON_NAME. HACKATHON_NAME is a DESCRIPTION hackathon spanning HACKATHON_LENGTH on HACKATHON_DATE.",
+      "Welcome to DesignVerse. Thank you for being on the organizing team, we appreciate your efforts to help support DesignVerse. DesignVerse is a DESCRIPTION hackathon spanning 24 hours on May 11, 2024.",
```

Comment:
we have the description nowww. I included in the description of your issue. Can you put that in for all the forms.